### PR TITLE
Remove Unnecessary Console Loggings

### PR DESCRIPTION
**Pull request by sdiazbarraza**
Hi! Good day everyone! 
I'm solve issue #68

### CODE REVIEW

**Review by king-11 — APPROVED** [review]

**Comment by king-11** [comment]
Hi @sdiazbarraza thanks for your contributions. 

We have one request that good first issues are meant for beginners if you fix them all we won't be able to accept new people as contributors 😅 so will have to close your other 2 PRs

File: context/auth.js [hunk 1]
```diff
@@ -22,11 +22,10 @@ export const AuthProvider = ({ children }) => {
         provider.addScope('profile');
         firebase.auth().signInWithPopup(provider)
             .then(() => {
-                //console.log("firebase login success")
+            //Notify to user by succes login
```
Comment (king-11) [inline context/auth.js:25]: Adding a modal would be a good option here

File: pages/hackathon/[slug]/index.js [hunk 2]
```diff
@@ -70,7 +69,7 @@ export default function Hackathon() {
                     setSubmisssions(response.data)
                 })
                 .catch((err) => {
-                    console.error(err)
+                    //Notify to user by the error
```
Comment (king-11) [inline pages/hackathon/[slug]/index.js:72]: Similarly, we can add a Modal here

**Review by shubhanshu02 — APPROVED** [review]
LGTM :+1:

**Review by king-11 — COMMENTED** [review]
I think the addition of Modal is of a higher priority now as without the console logs for login there is no way to tell the user what will happen.

File: pages/hackathon/[slug]/index.js (hunk 2 above)
Comment (shubhanshu02) [reply pages/hackathon/[slug]/index.js:72]: This is when the hackathon details page has loaded, but the submissions couldn't be loaded. So, maybe showing the error only on submissions tab would be better :thinking:

File: context/auth.js (hunk 1 above)
Comment (shubhanshu02) [reply context/auth.js:25]: Ya, a small modal to show the user has logged in/signed up successfully would be great.

**Pull request merged**